pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -1378,6 +1378,12 @@ struct LowerTileToPXAPass : public LowerTileToPXABase<LowerTileToPXAPass> {
                             OperandsAre<EltwiseFloat>>,
         EltwiseOpConversion<tile::ATanOp, StdOp<stdx::ATanOp>,
                             OperandsAre<EltwiseFloat>>,
+        EltwiseOpConversion<tile::ASinHOp, StdOp<stdx::ASinHOp>,
+            OperandsAre<EltwiseFloat>>,
+        EltwiseOpConversion<tile::ACosHOp, StdOp<stdx::ACosHOp>,
+            OperandsAre<EltwiseFloat>>,
```

Comment:
Could you please sort this

Reply:
@flaub do we want this sorted by type and then alphabetically (so keep all the trig functions from `CosOp` through `ATanHOp` together, but sorted alphabetically)? Or do we want to sort all the `EltwiseOpConversion` alphabetically (so e.g. the trig & comparison ops are mixed together)?

Reply:
Looks like this list of types weren't sorted to begin with, rather they are grouped. So I'll retract my request to have this one be sorted.

Reply:
Should I revert the order to by type first  and then by alphabet order for this?
I noticed there are some other files may also have this order issue, like in [activation.cpp](https://github.com/plaidml/plaidml/pull/1596/files#diff-ad29beaaa6bb56bd3e13199aa9ab56dcbe487fe408dfa71a9828bf36a175c1c8), [edsl_test.cc](https://github.com/plaidml/plaidml/pull/1596/files#diff-55aa61b72febc9ab6e112cb4e26668c807521b41def522f17aada5a7d047b9dd), it seems not strictly follow type or alphabet order.

Reply:
Type then alphabet is better than completely unsorted, so thanks for that change!